<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextPreviewWidget</class>
 <widget class="QWidget" name="TextPreviewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Preview Text:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="txt_preview_text">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Preview Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_variation">
       <property name="text">
        <string>Variation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_variation"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_background_color">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Background:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorPickerWidget" name="background_color_picker" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The background color of the text preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_resolution">
       <property name="text">
        <string>&lt;Resolution&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerWidget</class>
   <extends>QWidget</extends>
   <header>widgets/ColorPickerWidget.hpp</header>
   <container>1</container>
   <slots>
    <signal>color_changed(QColor)</signal>
    <signal>on_color_changed(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/UIResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>background_color_picker</sender>
   <signal>color_changed(QColor)</signal>
   <receiver>TextPreviewWidget</receiver>
   <slot>background_color_changed(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_variation</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TextPreviewWidget</receiver>
   <slot>on_font_variation_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>background_color_changed(QColor)</slot>
  <slot>on_font_variation_changed(int)</slot>
 </slots>
</ui>
